src/NaturalLanguage/Enums.cs

```diff
@@ -67,6 +67,8 @@ public enum NLTokenizerAttributes : ulong {
 	[iOS (12,0), Mac (10,14, onlyOn64: true), TV (12,0), Watch (5,0)]
 	public enum NLLanguage {
 		[DefaultEnumValue]
+		[Field (null)]
+		Unevaludated,
```

Comment:
Typo: `Unevaluated`

Reply:
good catch, I wonder why the typo check did not hit it locally ?!?